module_report/import_report_base.txt

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
If this and the other `txt` file you added in this directory are auto-generated from the code above, I'd say we should remove them because they can easily be generated again.

In other words, I don't think they need to live here.